ocaml/xapi/rbac_audit.ml

```diff
@@ -222,7 +222,7 @@ let zip data = (* todo: remove i/o, make this more efficient *)
            (fun fd ->
               Gzip.compress fd (fun fd ->
                   let len = String.length data in
-                  let written = Unix.write fd data 0 len in
+                  let written = Unix.write fd (Bytes.of_string data) 0 len in
```

Comment:
Another opportunity to use `Unix.write_substring`

Reply:
Done!